File: eval/fixtures/legacy-directory-backed-aspnet-commerce/src/Legacy/Customers/CustomerProfileController.cs
namespace Acme.Legacy.Customers;

[ApiController]
[Route("api/v6/customers")]
public class CustomerProfileController : ControllerBase
{
  private readonly CustomerProfileService _customerProfileService = new CustomerProfileService();

  [HttpGet("profiles/{customerId}/risk")]
  public IActionResult ReviewRisk(string customerId)
  {
    return Ok(_customerProfileService.ReviewProfileRisk(customerId));
  }
}

File: eval/fixtures/legacy-directory-backed-spring-commerce/src/main/java/com/acme/legacy/customers/CustomerProfileController.java
package com.acme.legacy.customers;

@RestController
@RequestMapping("/api/v5/customers")
public class CustomerProfileController {
  private final CustomerProfileService customerProfileService = new CustomerProfileService();

  @GetMapping("/profiles/{customerId}/risk")
  public ResponseEntity<?> reviewRisk(String customerId) {
    return ResponseEntity.ok(customerProfileService.reviewProfileRisk(customerId));
  }
}
